client/verta/verta/_internal_utils/_utils.py

```diff
@@ -68,19 +72,89 @@ def __init__(self, scheme=None, socket=None, auth=None, max_retries=0, ignore_co
             on HTTP codes {502, 503, 504} which commonly occur during back end connection lapses.
         ignore_conn_err : bool, default False
             Whether to ignore connection errors and instead return successes with empty contents.
-
+        credentials : :class:`~verta._internal_utils.credentials.EmailCredentials` or :class:`~verta._internal_utils.credentials.JWTCredentials`, optional
+            Either dev key or JWT token data to be used for authentication
+        headers: dict, optional
+            Additional headers to attach to requests
```

Comment:
[The styleguide](https://numpydoc.readthedocs.io/en/latest/format.html) puts a newline before the closing `"""`.
```suggestion
            Additional headers to attach to requests.

```